packages/calcite-components/src/components/dialog/dialog.scss

```diff
@@ -187,13 +187,15 @@ calcite-panel {
  * Sizes
  */
 @mixin dialog-size($size, $width) {
-  :host([width-scale="#{$size}"]) .dialog {
+  :host([width-scale="#{$size}"]) .dialog,
+  :host([width="#{$size}"]) .dialog {
```

Comment:
Not sure what's a good way to set this up. If I set one of these as a default "m", it overrides the other one set to a different value. If I leave the default out so they have equal weight, it takes the full width of the screen when none are set. How do we deprecate mindfully?